roofit/roofitcore/inc/RooFit/Detail/MathFuncs.h

```diff
@@ -414,10 +414,11 @@ inline double cbShape(double m, double m0, double sigma, double alpha, double n)
    if (t >= -absAlpha) {
       return std::exp(-0.5 * t * t);
    } else {
-      double a = std::pow(n / absAlpha, n) * std::exp(-0.5 * absAlpha * absAlpha);
+      // double a = std::pow(n / absAlpha, n) * std::exp(-0.5 * absAlpha * absAlpha);
+      double a = std::exp(-0.5 * absAlpha * absAlpha);
       double b = n / absAlpha - absAlpha;
 
-      return a / std::pow(b - t, n);
+      return a * std::pow(n / absAlpha / (b - t), n);
```

Comment:
`n / absAlpha` appears twice in each code path, and divisions are relatively expensive.

Maybe define a `double r = n / absAlpha` before and reuse that?